class.jetpack-gutenberg.php

```diff
@@ -304,7 +320,7 @@ public static function preset_exists( $preset ) {
 	public static function get_preset( $preset ) {
 		return json_decode(
 			// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
-			file_get_contents( JETPACK__PLUGIN_DIR . self::get_blocks_directory() . $preset . '.json' )
+			file_get_contents( JETPACK__PLUGIN_DIR . self::get_blocks_source_directory() . 'setup/' . $preset . '.json' )
```

Comment:
Is there a better name than "preset" now that the concept of building individual blocks is gone?